Installation/CHANGES.md

```diff
@@ -32,6 +32,12 @@ Release date: December 2020
       As explained above, the code is backward compatible. However, it is recommended
       that all calls to `decompose()` are fixed to use the new interface.
 
+### [Surface Mesh](https://doc.cgal.org/5.2/Manual/packages.html#PkgSurfaceMesh)
+
+-   Added the function `clear_without_removing_property_maps()` to clear a mesh but keep all the created property maps added.
+-   Added the functions `remove_property_maps()` and `remove_all_property_maps()` to remove all added property maps by index type or all of them respectively.
```

Comment:
`remove_property_maps` is a function template, and not a function. I suggest to explicit that using:

```suggestion
-   Added the functions `remove_property_maps<Index_type>()` and `remove_all_property_maps()` to remove all added property maps by index type or all of them respectively.
```

(I want to say, again, that it is a bad idea to discrimine on index types. It should be allowed that all index types are identical (for example `int`)/ Instead, it should be different names for functions, or using tags to discriminate on element types.)

Reply:
The different index types were introduced  for more type safety.

Reply:
@lrineau this is a pure `Surface_mesh` function and the index types are different. Index type here mean simplex type.